src/cascadia/WindowsTerminal/AppHost.cpp

```diff
@@ -57,6 +57,8 @@ AppHost::AppHost() noexcept :
         _window = std::make_unique<IslandWindow>();
     }
 
+    _window->IsQuakeWindow(_logic.IsQuakeWindow());

```

Comment:
Should we be calling `_IsQuakeWindowChanged` instead?

Reply:
Nah, because `_IsQuakeWindowChanged` assumes that the window actually already exists. This is before the HWND has actually been created. `_HandleCreateWindow` will do the right thing for quake mode when the window is first created.

Reply:
Ignore me, I was thinking of `_enterQuakeMode`. Yea sure we can do _IsQuakeWindowChanged